### PR TITLE
Here I tried to add the installer package for the C/C++ compiler MinG…

### PR DESCRIPTION
**Pull request by ritchielrez**
…W, so everyone can download this package through Winget easily.

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate <manifest>`, where `<manifest>` is the name of the manifest you're submitting?
- [x] Have you tested your manifest locally with `winget install -m <manifest>`?

-----

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/2938)

### CODE REVIEW

**Comment by ritchielrez** [comment]
I hope this will turn out good. I've tried out my best.

**Review by RitchielRez1 — APPROVED** [review]
**Good** **job!**

**Review by ritchielrez — COMMENTED** [review]
**Thanks!**

File: manifests/Ritchiel/MinGW/0.6.3.0.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+AppMoniker: g++
+Tags: gcc,g++
+Description: MinGW, a contraction of "Minimalist GNU for Windows", is a minimalist development environment for native Microsoft Windows applications.
+Homepage: http://www.mingw.org/
+Installers:
+  - Arch: x86
+    InstallerType: EXE
+    Url: https://osdn.net/frs/redir.php?m=xtom_us&f=mingw%2F68260%2Fmingw-get-setup.exe
+    Sha256: a5de8c70676fdd157414e3116c7cb6f3dba48e549b5880177b10e4e6bc6c282e
+# Generated by https://github.com/ptorr-msft/WinGetYamlGenerator
```
Comment (RitchielRez1) [inline manifests/Ritchiel/MinGW/0.6.3.0.yaml:17]: **_Seems a not a bad concept!_**

**Review by RitchielRez1 — APPROVED** [review]
### **Wonderful  contribution!**

**Comment by RitchielRez1** [comment]
> Wonderful  contribution!

Approved!

**Comment by ritchielrez** [comment]
I've tried to add **Silent** and **SilentWithProgress** switches but none of them worked for the setup.

**Comment by denelon** [comment]
@ritchielrez I did some searching and I found a very old reference to a request to support a silent install for minGW.
https://sourceforge.net/p/mingw/bugs/2176/

I'm not sure if there is a way to provide a scripted install with default options applied or not. That issue is quite old.

We also don't have support for filtering or preferring silent install which is why we require silent in the community repository.
https://github.com/microsoft/winget-cli/issues/910

Once that issue has been implemented, we may consider lifting that restriction. Users would be able to avoid interactive installers for their automated install processes like import.
https://github.com/microsoft/winget-cli/issues/910

**Comment by denelon** [comment]
The manifest schema has also changed. This needs to be updated to the v1.0 schema.

**Comment by ritchielrez** [comment]
I don't use Windows anymore. If anyone can resolve this issue it would be a contribution.

**Pull request closed without merging**